src/lib/pka.c

```diff
@@ -274,12 +280,12 @@ pka_instance_t pka_init_global(const char *name,
                                uint32_t    cmd_queue_size,
                                uint32_t    result_queue_size)
 {
-    uintptr_t       shmem_addr;
-    uint32_t        shmem_size;
-    uint8_t        *shmem_ptr;
-    char            shmem_name[PKA_SHMEM_NAME_SIZE];

```

Comment:
Any reason why we are using dynamically allocated memory versus statically allocated memory?

Reply:
Yes, the statically allocated memory is local and assigned to a pointer `pka_gbl_info->shmem_info.name` later in this function. This will be used later on when removing the shared memory in `pka_term_global()`. But, because this is local, the array contents cannot be accessed later on with just the pointer.

Reply:
Sorry my point is, we could've define the shmem_name as global instead of defining it in the stack.
But no worries, if you did it this way, it is fine too ;)